### PR TITLE
Banners

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

**Review by EnzoVieira — CHANGES_REQUESTED** [review]

File: app/page.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import Hero from "@/components/Hero";
+import HeroSection from "@/components/HeroSection";
 
 export default function Home() {
   return (
     <main className="bg-background px-20">
       <Hero />
+      <HeroSection />
```
[the remark this answers is not in the document]
Comment (GresosJ) [reply app/page.tsx]: Done.

File: components/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            <div
+              key={colIndex}
+              className={`relative ${
+                rowIndex === 0 && colIndex === 0
+                  ? "col-span-2"
+                  : rowIndex === 1
+                  ? "col-span-4"
+                  : rowIndex === 2
+                  ? "col-span-2"
+                  : "col-span-1"
```
[the remark this answers is not in the document]
Comment (GresosJ) [reply components/HeroSection.tsx]: Done.

File: components/Banners.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  if (rowIndex === 0) {
+    return colIndex === 0 ? 2 : 1;
+  } else if (rowIndex === 1) {
+    return 4;
+  } else if (rowIndex === 2) {
+    return 2;
+  } else {
+    return 1;
+  }
+}
```
Comment (EnzoVieira) [inline components/Banners.tsx:25]: tailwind advises against constructing class names dynamically, instead, return the complete class names.

see [https://tailwindcss.com/docs/content-configuration#dynamic-class-names](url)

**Review by EnzoVieira — CHANGES_REQUESTED** [review]

File: components/Banners.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import Link from "next/link";
+import React from "react";
+
+const items: string[][] = [
```
Comment (MarioRodrigues10) [inline components/Banners.tsx:4]:
```suggestion
type LinkGroup = string[];
const items: LinkGroup[] = [
```

File: components/Banners.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  ["Ser Parceiro", "Parceiros", "Equipa"],
+  ["Sobre Nós"],
+  ["Departamento de Informática", "Estudar LEI"],
+];
+
+interface RowProps {
+  links: string[];
```
Comment (MarioRodrigues10) [inline components/Banners.tsx:11]:
```suggestion
  links: LinkGroup;
```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Pull request merged**